packages/arb-token-bridge-ui/src/hooks/useNetworks.ts

```diff
@@ -58,6 +63,26 @@ export function sanitizeQueryParams({
   sourceChainId: ChainId | number
   destinationChainId: ChainId | number
 } {
+  // early check, if we detect an orbit chain in the route
+  // if neither of source or destination chains are the orbit chain in route, return the orbit chain as the destination
+  if (
+    sourceChainId === undefined &&
+    destinationChainId !== undefined &&
+    isNetwork(destinationChainId).isOrbitChain
+  ) {
+    const orbitChain = getOrbitChains().find(
+      chain => chain.chainID === destinationChainId
+    )
+    const [defaultSourceChainId] = getDestinationChainIds(destinationChainId)
+    return {
+      sourceChainId:
+        getTeleportChainIdForOrbitChain(destinationChainId) ?? // show the grand-parent-chain id if valid, else get the parent chain id
```

Comment:
I'd rather we default to L2 being source here, that's the flow that gets more usage

Reply:
yah sgtm to default to Arb One -> L3 instead of Ethereum -> L3

Reply:
Updated.